src/planning/planner.ts

```diff
@@ -227,15 +227,15 @@ export class Planner implements InspectablePlanner {
     let relevance: Relevance|undefined = undefined;
     let description: Description|null = null;
     if (this._shouldSpeculate(plan)) {
-      log(`speculatively executing [${plan.name}]`);
+      //log(`speculatively executing [${plan.name}]`);
```

Comment:
I left this commented out because I'd like to put it back after I work out why it was logging during tests (which I don't expect). Same for line 238.